### ISSUE TITLE
Printer not Restarting

### ISSUE THREAD

**Issue by gizzburn** [TheZeroBeast/MM-control-01#158]
Ensure you're running the latest **RELEASE** and you've checked the **Wiki** for answers before reporting. Any issues reported regarding previous **RELEASES** will automatically be closed and not responded to, this ensures the developer maintains focus & streamlines the development process.

**TZB-MMU2S Release Version**
The version of the release relating to the bug report.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Serial Log**
Please attach and serial logs of this issue.

**Additional context**

Hi Robert, I have just tried to install the TZB-MK3S-3.0.0-BN2462-Bondtech-16.hex firmware to my printer, it installs ok but the printer never restarts, it just sits there with the onscreen message verifying 100%.
Tried it a few times but with the same results.
Printer is a MK3s + MMU2s with Bondtech BMG extruder.
Any ideas please..

**Comment by TheZeroBeast** [comment]
Hey @gizzburn,
Did you get the MMU2S software on as well as until it’s on the printer side won’t boot.

**Comment by gizzburn** [comment]
Ah no I did not Robert will try again and do both this time lol..

**Comment by TheZeroBeast** [comment]
https://github.com/TheZeroBeast/TZB-MMU2S-Firmware/wiki/Flashing-Procedure
Video I made on the flashing 👍

**Comment by gizzburn** [comment]
I even watched that Robert as well, old age .

**Comment by TheZeroBeast** [comment]
Yeah, I know that feeling 👍
Let me know if it all works with the bondtech bmg👍

**Comment by gizzburn** [comment]
Will do

**Comment by gizzburn** [comment]
Still no go Robert, the printer does not reboot after the update, I managed to get the MMU2s updated, but still the printer does not boot up fully, it justs sits there with 100% verifying

**Comment by TheZeroBeast** [comment]
That is even after power cycling the printer I take it?
Interesting.... Let me have a think why that would be.

**Comment by gizzburn** [comment]
Yes power it off and on makes no difference

**Comment by TheZeroBeast** [comment]
Hi @gizzburn,

I think I found what happened, I've recompiled below.
Can you confirm that the BMG requires the below changes, as these are what are made in this config.

``` #define CUSTOM_MENDEL_NAME "Bondtech-PE MK3s"
#define DEFAULT_AXIS_STEPS_PER_UNIT {100,100,3200/8,415}
#define Z_MAX_POS 205
#define TMC2130_USTEPS_E 16
#define LOAD_FILAMENT_1 "G1 E80 F400"
#define UNLOAD_FILAMENT_1 "G1 E-95 F7000"
#define FILAMENTCHANGE_FINALRETRACT -95
#define FILAMENTCHANGE_FIRSTFEED 80
```


[TZB-MK3S-3.0.1-RC1-BN2463-Bondtech-16.hex.zip](https://github.com/TheZeroBeast/TZB-MMU2S-Firmware/files/3707351/TZB-MK3S-3.0.1-RC1-BN2463-Bondtech-16.hex.zip)

**Comment by gizzburn** [comment]
Success Robert will run some tests now after No1 Calibration, thanks a lot...

**Comment by gizzburn** [comment]
Robert just done a test cube,infact done two, one with the Bondtech firmware and one with your firmware,both cubes done on identical settings. I noticed on the Bondtech firmware the cube shows lines, I first noticed them on a lithopane I did, I printed the same one before the Bondtech firmware upgrade for a comparison.
But on your firmware those lines are much less visable, great work and thanks for your great Firmware.
